src/app/services/geo.service.ts

```diff
@@ -48,7 +48,7 @@ export class GeoService {
   getLocations(radius: number, coords: Array<number>) {
     const query: GeoQuery = this.locations.near({
       center: new firestore.GeoPoint(coords[0], coords[1]),
-      radius: radius,
+      radius,
```

Comment:
Is this working properly?

Reply:
yes, this is not a problem and will work as it should. https://eslint.org/docs/rules/object-shorthand